git-link.el

```diff
@@ -35,6 +35,10 @@
 
 ;;; Change Log:
 
+;; 2022-12-27 - v0.8.7
+;; * Add support for [gitiles](https://gerrit.googlesource.com/gitiles) (Issue #100)
+;; * Add support for googlesource.com using gitiles
```

Comment:
This will get added when v0.8.7 —or whatever version— is made. No need to add it here.

Reply:
Sounds good, I'll remove it :+1: